### ISSUE TITLE
File not found when import .less file in a .less file required by .tsx file

### ISSUE THREAD

**Issue by farawaaay** [egoist/rollup-plugin-postcss#68]
Problem:
I have a .tsx file which imports a .less style file. In this .less file, I `@import 'another/file/like/variable.less'`, but I only get `File not found` error when I compile with rollup.

rollup.config.js

```javascript
import resolve from 'rollup-plugin-node-resolve';
import postcss from 'rollup-plugin-postcss';
import typescript from 'rollup-plugin-typescript2';
import filesize from 'rollup-plugin-filesize';
import progress from 'rollup-plugin-progress';
import cleaner from 'rollup-plugin-cleaner';
import commonjs from 'rollup-plugin-commonjs';
import path from 'path';
// import fs from 'fs';

const config = {
  input: './src/index.ts',
  output: {
    file: './lib/index.js',
    format: 'es',
    banner: '/* eslint-disable */',
  },
  plugins: [
    // rollup-plugin-cleaner
    cleaner({
      targets: ['./lib'],
    }),
    // rollup-plugin-postcss
    postcss({
      inject: false,
      extract: true,
      minimize: false,
      modules: true,
      to: 'lib/index.css',
      plugins: [
        require('autoprefixer')(),
      ],
    }),
    // rollup-plugin-node-resolve
    resolve(),
    // rollup-plugin-typescript2
    typescript(),
    // rollup-plugin-filesize
    filesize(),
    // rollup-plugin-progress
    progress(),
    // rollup-plugin-commonjs
    commonjs(),
  ],
  external: [
    'react',
    'react-dom',
  ],
};

export default config;
```

tsconfig.json

```json
{
  "include": [
    "src",
  ],
  "exclude": [
    "node_modules"
  ],
  "compilerOptions": {
    "target": "es5",                          
    "module": "es2015",                    
    "lib": ["es2015", "es2015.core", "dom"],                            
    "jsx": "react",
    "declaration": true,                 
    "removeComments": true,      
    "strict": true,                           
    "moduleResolution": "node",           
    "typeRoots": ["./typings", "./node_module/@types"],                       
    "types": ["modules", "node"],                           
    "allowSyntheticDefaultImports": true
  }
}

```

file in ./typings
```typescript

declare module "*.css" {
    const content: {
        [key: string]: string
    };
    export default content;
}

declare module "*.less" {
    const content: {
        [key: string]: string
    };
    export default content;
}

```

But when I run `rollup -c`, I get error:
😨
```
./src/index.ts → ./lib/index.js...
(4): node_modules/tslib/tslib.es6.js[!] (postcss plugin) Error: '../../../styles/variables.less' wasn't found. Tried - ../../../styles/variables.less,../../../styles/variables.less
src/components/Common/Portal/style.less
```

In my index.ts, I import a ts file like:

```typescript
import styles from './style.less';

console.log(styles);
```

In the './style.less' file:
```less
@import "../../../styles/variables.less";

.test-less {
  background: @background;
}
```

Any help is appreciated! 😁
(And sorry for my bad English! 😅)

**Comment by egoist** [comment]
Hmm just checked out how gulp-less works and it seems we need to pass `filename` to less options: https://github.com/egoist/rollup-plugin-postcss/blob/beef7c55a4f848a419693b9c91a723e661618e6c/src/less-loader.js#L10-L12

Didn't find any related docs on lesscss.org tho 😕

**Comment by farawaaay** [comment]
OK, I will help fix it. 🤔

**Comment by DongShelton** [comment]
This issue seems like that doesn't use less correctly, namely, miss the filename option.🤔
You could see the issue [File not found when @import relative path](https://github.com/less/less.js/issues/3166). 
And see the docs [http://lesscss.org/usage/#programmatic-usage](http://lesscss.org/usage/#programmatic-usage).
So, I sent a PR for fix this problem: [fix: fix less file not found](https://github.com/egoist/rollup-plugin-postcss/pull/74).
By the way, why I am unable to pass the test?😨😨😨

**Comment by egoist** [comment]
It should be fixed now in `v1.2.9`, let me know if it's working for you.
